mqtt_exporter.py

```diff
@@ -455,10 +455,23 @@ def clear(self):
 
     def update(self, label_values, value):
         child = self.metric.labels(*label_values)
+
+        # Check if value is a string that looks like a JSON object
+        if isinstance(value, str) and value.startswith('{') and value.endswith('}'):
+            try:
+                parsed_value = json.loads(value)
+                if 'value' in parsed_value:
```

Comment:
This is not very flexible. I have sensors returning Json which is not a flat dict, but has some hierarchy, where this approach is not working. See example above.